devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -237,14 +245,19 @@ def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, ep = null)
 	changeSwitch(ep, cmd)
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicSet cmd, ep = null) { 
+	log.debug "Basic ${cmd}" + (ep ? " from endpoint $ep" : "")
+	[changeSwitch(ep, cmd), response(["delay 2000", encap(zwave.meterV3.meterGet(scale: 2), endpoint)])]
+}
+
 def zwaveEvent(physicalgraph.zwave.commands.switchbinaryv1.SwitchBinaryReport cmd, ep = null) {
 	log.debug "Binary ${cmd}" + (ep ? " from endpoint $ep" : "")
 	changeSwitch(ep, cmd)
 }
 
 private changeSwitch(endpoint, cmd) {
 	def value = cmd.value ? "on" : "off"
-	if (endpoint == 1) {
+	if (endpoint == 1 || endpoint == null) {
```

Comment:
giving it more thought, I'd also make only one comparison here, depending on a device's model. There's a possibility that in Qubino Flush 2 Relay's case, root endpoint contains summed up meter values, and we want to avoid them being treated as reports of first output.